src/dso_hdr.cc

```diff
@@ -344,8 +343,7 @@ DsoFindRes DsoHdr::insert_erase_overlap(DsoMap &map, Dso &&dso) {
   _stats.incr_metric(DsoStats::kNewDso, dso._type);
   LG_DBG("[DSO] : Insert %s", dso.to_string().c_str());
   // warning rvalue : do not use dso after this line
-  return map.insert(std::make_pair<ProcessAddress_t, Dso>(
-      ProcessAddress_t(dso._start), std::move(dso)));
+  return map.insert(std::pair(ProcessAddress_t(dso._start), std::move(dso)));
```

Comment:
In this case, I might have kept the make_pair. Though I'm not sure it would change anything